src/Whim/Layout/FreeLayoutEngine.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System.Linq;
+using DotNext.Collections.Generic;
```

Comment:
Could this line be removed?

Reply:
Yes, I was trying stuff, and it was added automatically on some stuff I removed.